app/models/loaned_book.rb

```diff
@@ -1,3 +1,6 @@
 class LoanedBook < ApplicationRecord
   enum status: {rending: 0, returned: 1 , overdated: 2}
+
+  belongs_to :users, class_name: User.name, foreign_key: :member_id
```

Comment:
```suggestion
  belongs_to :user, foreign_key: :member_id
```
suggest nên set field là user_id luôn thay vì member_id

Reply:
Nếu em đổi thành user_id thì em có thể rút ngắn đoạn code này phải không chị hè ?